src/Pages/HomePage.jsx

```diff
@@ -1,110 +1,106 @@
-    }
-    else{
+    } else {
       ref.current.classList.remove('slide__in');
       ref2.current.classList.remove('slide__in2');
       ref3.current.classList.remove('slide__in3');
-      ref4.current.classList.remove('slide__in');
     }
-  })
-
+  });
 
   // Intersection Observer [I]
   useEffect(() => {
-    const observer= new IntersectionObserver(([entry])=>{
+    const observer = new IntersectionObserver(([entry]) => {
       setIsVisible(entry.isIntersecting);
       console.log(entry.isIntersecting);
-    })
+    });
     observer.observe(ref.current);
-    return ()=> observer.disconnect();
-},[]);
+    return () => observer.disconnect();
+  }, []);
 
   return (
-    <div
-      style={{
-        background:
-          'linear-gradient(31.67deg, rgba(3, 14, 21, 0.96) -2.58%, #05121B 70.47%)',
-      }}
-      className='h-screen overflow-y-scroll'
-    >
+    <div className='flex flex-col max-h-screen min-h-screen overflow-auto bg-gradient-to-r from-slate-950 to-gray-900'>
       <Navbar />
 
-      <div  className=' relative lg:flex-row pt-12 md:pt-20 space-y-14 lg:space-y-0  px-6 flex flex-col-reverse lg:px-20 py-8 justify-between'>
-        <div className='hidden lg:block absolute top-0 right-0'>
-          <img src={bg} className='h-48 w-80' />
-        </div>
+      <div className='relative px-6 py-8 pt-12 grow'>
         {/* headline and subhead... */}
-        <div  className='flex flex-col space-y-10  '>
-          <div className='home__image' ref={ref}>
-            <h1 className='text-[#FFFFFF] text-4xl md:text-6xl lg:text-7xl font-semibold'>
-              Discover.Explore.
+        <div className='flex flex-col items-center space-y-6 grow'>
+          <div className='flex flex-col items-center justify-center w-full'>
+            <h1
+              ref={ref}
+              className='inline-block font-medium text-transparent sm:text-6xl md:text-7xl text-5xl lg:text-8xl text-center bg-gradient-to-r from-white to-neutral-400 bg-clip-text w-4/5 m-0'
+            >
+              <span className='inline'>Discover </span>
+              <span className='inline'> Explore </span>
+              <span className='block'>& Share</span>
             </h1>
-            <p className='text-[#FFFFFF] text-4xl md:text-6xl lg:text-7xl font-semibold'>
-              Share.
-            </p>
           </div>
-          <p  ref={ref2} className='home__image text-[#FFFFFF]  text-lg md:text-xl leading-8 max-w-2xl'>
-            <span className='text-[#6F4FF2] font-normal '>SnapGrid</span> is a
-            versatile web platform that simplifies image discovery and
-            downloads. Users can effortlessly find and acquire high-quality
-            images spanning various categories, from nature to architecture.
+
+          <p
+            ref={ref2}
+            className='w-4/5 max-w-2xl leading-8 text-center text-md text-neutral-400 md:text-xl'
+          >
+            Used by some of the world's largest companies,{' '}
```

Comment:
Remove this as we don't right now used by companies.

Reply:
ok